.prettierrc.json

```diff
@@ -0,0 +1,9 @@
+{
+    "trailingComma": "es5",
+    "tabWidth": 2,
+    "semi": true,
+    "singleQuote": true,
+    "editor.formatOnSave": true,
+    "bracketSpacing": true,
+    "printWidth": 200
+  }
```

Comment:
@paslavskyi9roman apply correct formating to this file

Reply:
@JimHolman1199 Fixed with this commit 31744f2a10a6a01538bc0bf0a91e7b9bd94a37b1